compiler/orchestrator_runtime/pash_init_setup.sh

```diff
@@ -87,11 +89,20 @@ do
     if [ "--distributed_exec" == "$item" ]; then
         export distributed_exec=1
     fi
+
+    if [ "--local-annotations-dir" == "$item" ]; then
+        count=$(( count + 1 ))
```

Comment:
no need to use count, see how other options are parsed! Like in line 73. We first set a variable that the next argument is the option argument, and then we check that argument in the next iteration of the loop.